### PR TITLE
[Storage] [DataMovement] Added tests for Share Directory to Blob Virtual Directory

### PR DESCRIPTION
**Pull request by amnguye**
Added tests for Share Directory to Blob Virtual Directory

### CODE REVIEW

File: sdk/storage/Azure.Storage.DataMovement.Files.Shares/BlobToFileSharesTests/ClientBuilderExtensions.cs [hunk 1]
```diff
@@ -21,6 +21,8 @@
 using Azure.Storage.Blobs.Models;
 using Azure.Storage.Test;
 using DMBlob::Azure.Storage.DataMovement.Blobs;
+using Azure.Core;
+using Azure.Core.TestFramework;
```
Comment (jaschrep-msft) [inline sdk/storage/Azure.Storage.DataMovement.Files.Shares/BlobToFileSharesTests/ClientBuilderExtensions.cs:25]: Only change in the file?

**Review by jaschrep-msft — APPROVED** [review]

File: sdk/storage/Azure.Storage.DataMovement.Files.Shares/BlobToFileSharesTests/ClientBuilderExtensions.cs (hunk 1 above)
Comment (amnguye) [reply sdk/storage/Azure.Storage.DataMovement.Files.Shares/BlobToFileSharesTests/ClientBuilderExtensions.cs:25]: Good catch, will remove this. I was trying to do some fun stuff with SAS and decided not to do it here.

**Comment by amnguye** [comment]
Slipped my mind to add Page and Append blob, will do that now.

**Pull request merged**